Tests/Fixtures/app/AppKernel.php

```diff
@@ -44,7 +44,17 @@ public function getLogDir()
 
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
-        $loader->load(__DIR__ . '/config/' . $this->environment . '.yml');
+        $loader->load(__DIR__ . '/config/symfony-'.self::getRoutingVersion(). '/' . $this->environment . '.yml');
+    }
+
+    private static function getRoutingVersion()
+    {
+        $installedPackages = json_decode(file_get_contents(__DIR__.'/../../../vendor/composer/installed.json'));
```

Comment:
Yes, this is ugly, but it is the only way to get it working.
Using `AppKernel::VERSION_MAYOR` unfortunately does not work, because `symfony/framework-bundle:2.8.*` pulls in `symfony/routing:~2.8|~3.0.0`, and `symfony/http-kernel:~2.8.*`